<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriendRequestWidget</class>
 <widget class="QWidget" name="AddFriendRequestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>593</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>593</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>申请加好友</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qqicon.qrc">
    <normaloff>:/qqicons/qrc/QQicon.svg</normaloff>:/qqicons/qrc/QQicon.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AddFriendRequestWidget{	
	background-color: rgb(250, 250, 250);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1000,1" columnstretch="1,1000,1">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="361,40">
     <property name="spacing">
      <number>40</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="30,15,50,15,18">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,20">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="lblHeadPic">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>头像</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="lblName">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>昵称</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblAccount">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	font-weight: 400;
	color: rgb(153, 153, 153);
}</string>
             </property>
             <property name="text">
              <string>账号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblAuthInfoTip">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	font-weight: 400;
	color: rgb(153, 153, 153);
}</string>
         </property>
         <property name="text">
          <string>填写验证信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="edtAuthInfo">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
	border: 1px solid grey;  
	border-radius: 8px;
	padding: 2px 2px 2px 2px;
}

QPlainTextEdit:focus{
	border-color: rgb(0, 153, 255);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblGroupTip">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	font-weight: 400;
	color: rgb(153, 153, 153);
}</string>
         </property>
         <property name="text">
          <string>分组</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbGroupSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
	border: 1px solid grey;
	border-radius: 10px;
	padding-left: 10px;/*左侧边距*/
	font: 12pt &quot;微软雅黑&quot;;
}

QComboBox:focus{
	border-color: rgb(0, 153, 255);
}

QComboBox::drop-down{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	border:none;
	margin-right: 20px;
}

QComboBox::down-arrow{
	image: url(:/qqicons/qrc/119-down-arrow.svg);
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="100,70,70">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 170, 255);
	border-radius: 5px;
}

QPushButton:hover{	
	background-color: rgb(0, 161, 242);
}

QPushButton:pressed{
	color: rgb(226, 226, 226);
	background-color: rgb(0, 151, 226);
}

QPushButton:disabled{
	color: rgb(255, 255, 255);
	background-color: rgb(179, 212, 255);
}</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid grey;
	border-radius: 5px;
}

QPushButton:hover{	
	background-color: rgb(233, 233, 233);
}

QPushButton:pressed{
	color: rgb(97, 97, 97);
	background-color: rgb(214, 214, 214);
}
</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>38</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qqicon.qrc"/>
 </resources>
 <connections/>
</ui>
